### PR TITLE
Review messages and tooltips for Preferences

### PR DESCRIPTION
**Pull request by Mrt134**

### CODE REVIEW

**Comment by Mrt134** [comment]
rebuild

File: src/app/pages/preferences/page/forms/customtheme/customtheme.component.ts [hunk 1]
```diff
@@ -96,72 +96,78 @@ export class CustomThemeComponent implements OnInit, OnChanges, OnDestroy {
...
-          type: 'input', 
-          name: 'label', 
+        {
+          type: 'input',
+          name: 'label',
           width:'100%',
           placeholder: 'Menu Label',
           required:true,
-          tooltip: 'Specify how the theme name should appear in the menu.',
+          tooltip: 'Enter how the theme name will appear in the menu.',
```
Comment (wblock) [inline src/app/pages/preferences/page/forms/customtheme/customtheme.component.ts]: It's not clear what this means.  "how the theme name will appear" could be about color or location, but I think this is just a name.  So the Name is the long name, and the Menu Label is the short name that appears in the menu?

File: src/app/pages/preferences/page/forms/custom-theme-manager-form.component.ts [hunk 2]
```diff
@@ -21,21 +21,21 @@ export class CustomThemeManagerFormComponent implements OnInit, OnChanges {
...
+   customThemes?: Theme[];
    favoriteThemes?: string[]; // Theme Names
    showTooltips:boolean; // Form Tooltips on/off
    metaphor:string; // Prefer Cards || Tables || Auto (gui decides based on data array length)
 
    */
 
   public themesExist:boolean = false;
-  public emptyMessage: string = "No custom themes to manage. Click below to create a new custom theme."
+  public emptyMessage: string = "No custom themes. Click 'Create New Theme' to create a new custom theme."
```
Comment (wblock) [inline src/app/pages/preferences/page/forms/custom-theme-manager-form.component.ts]: Should this be <i></i>, or can HTML be used here?

File: src/app/pages/preferences/page/forms/customtheme/customtheme.component.ts [hunk 3]
```diff
@@ -96,72 +96,78 @@ export class CustomThemeComponent implements OnInit, OnChanges, OnDestroy {
...
+        {
+          type: 'select',
+          name: 'labelSwatch',
           width:'100%',
-          placeholder: 'Menu Swatch', 
+          placeholder: 'Menu Swatch',
           required:true,
           options:this.colorOptions,
-          tooltip: "Choose which color from the palette will be used for the label swatch that appears left of the label in the menu.",
+          tooltip: "Choose which color from the palette will be used for \
```
Comment (wblock) [inline src/app/pages/preferences/page/forms/customtheme/customtheme.component.ts]: This is confusing, probably due to the use of directions.  Maybe
"Choose the color to display next to the label in the menu."

File: src/app/pages/preferences/page/forms/customtheme/customtheme.component.ts [hunk 4]
```diff
@@ -96,72 +96,78 @@ export class CustomThemeComponent implements OnInit, OnChanges, OnDestroy {
...
+        {
+          type: 'select',
+          name: 'primary',
           width:'100%',
-          placeholder: 'Choose Primary', 
+          placeholder: 'Choose Primary',
           required:true,
           options:this.colorOptions,
-          tooltip: "Choose which color from the palette will be the theme's primary color",
+          tooltip: "Choose a color from the palette to be primary theme \
```
Comment (wblock) [inline src/app/pages/preferences/page/forms/customtheme/customtheme.component.ts]: "Choose the primary color for the theme."

File: src/app/pages/preferences/page/forms/customtheme/customtheme.component.ts [hunk 5]
```diff
@@ -96,72 +96,78 @@ export class CustomThemeComponent implements OnInit, OnChanges, OnDestroy {
...
+        {
+          type: 'select',
+          name: 'accent',
           width:'100%',
-          placeholder: 'Choose Accent', 
+          placeholder: 'Choose Accent',
           required:true,
           options:this.colorOptions,
-          tooltip: "Choose which color from the palette will be the theme's accent color",
+          tooltip: "Choose a color from the palette to be the accent \
```
Comment (wblock) [inline src/app/pages/preferences/page/forms/customtheme/customtheme.component.ts]: "Choose the accent color for the theme."

File: src/app/pages/preferences/page/forms/customtheme/customtheme.component.ts [hunk 6]
```diff
@@ -510,17 +532,17 @@ export class CustomThemeComponent implements OnInit, OnChanges, OnDestroy {
         // Check for duplicate theme names and labels
         let dupeName = this.isDuplicateOf("name",data.name);
         if(dupeName){
-          messages.push("This theme name is already taken. Please choose another name.")
+          messages.push("This theme name is in use. Enter another name.")
```
Comment (wblock) [inline src/app/pages/preferences/page/forms/customtheme/customtheme.component.ts]: "This name is already being used by an existing theme. Please choose a new name."

File: src/app/pages/preferences/page/forms/customtheme/customtheme.component.ts [hunk 7]
```diff
@@ -510,17 +532,17 @@ export class CustomThemeComponent implements OnInit, OnChanges, OnDestroy {
...
         }
 
         let dupeLabel = this.isDuplicateOf("label",data.label);
         if(dupeLabel){
-          messages.push("This theme label is already taken. Please choose another label.")
+          messages.push("This theme label is in use. Choose another label.")
```
Comment (wblock) [inline src/app/pages/preferences/page/forms/customtheme/customtheme.component.ts]: "This label is already being used by an existing theme. Please choose a new label."

**Comment by Mrt134** [comment]
rebuild

**Pull request merged**